### PR TITLE
dnsdist cli option improvements

### PR DESCRIPTION
**Pull request by zeha**
### Short description
Tiny improvements to the available dnsdist program options, and the --help and --version output.

Intended to be on top of #6433, might need a rebase then.

### Checklist
I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled and tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by Habbie** [comment]
> Intended to be on top of #6433, might need a rebase then.

Please feel free to include 6433 in this one.

**Comment by zeha** [comment]
Includes #6433 now!

File: pdns/dnsdist.cc [hunk 1]
```diff
@@ -2082,9 +2084,9 @@ try
   string optstring;
   for(;;) {
 #ifdef HAVE_LIBSODIUM
-    int c=getopt_long(argc, argv, "a:hcde:C:k:l:vp:g:u:V", longopts, &longindex);
+    int c=getopt_long(argc, argv, "a:hce:C:k:l:v:g:u:V", longopts, &longindex);
```
Comment (rgacogne) [inline pdns/dnsdist.cc]: `--verbose` doesn't take any argument so the `:` after the `v` should go (same thing below).

Comment (zeha) [reply pdns/dnsdist.cc]: Fixed.

**Comment by zeha** [comment]
plus more cleanup...

**Pull request merged**